indra/llcommon/StackWalker.h

```diff
@@ -103,7 +103,7 @@ class StackWalker
   StackWalker(DWORD dwProcessId, HANDLE hProcess);
   virtual ~StackWalker();
 
-  typedef BOOL (__stdcall *PReadProcessMemoryRoutine)(
```

Comment:
I'm pretty sure these cannot be changed, since they are not our BOOL, but rather the WinAPI BOOL from windows.h so changing these would be an ABI break.

this is another reason to move our code away from using the LL BOOL definition, but we need to be extremely careful to separate these out properly

Reply:
Good point. I will remove the StackWalker files from the commit.

Reply:
Done :)